benchmark/cvar_benchmark.py

```diff
@@ -15,16 +15,8 @@
 import numpy as np
 import scipy.sparse as sp
 
-from benchmark.benchmark_base import Benchmark
-
-
-class CVaRBenchmark(Benchmark):
-    @staticmethod
-    def name() -> str:
-        return "CVaR"
-
-    @staticmethod
-    def get_problem_instance() -> cp.Problem:
+class CVaRBenchmark():
+    def setup(self):
```

Comment:
Some benchmarks were still failing for me, I suggest we set the time out very high for now.
I recommend using `asv run --quick` for developing.

```suggestion
class CVaRBenchmark():
    timeout = 999

    def setup(self):
```